src/index.html

```diff
@@ -1,14 +1,80 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="navigation">
+      <div class="navigation__wrapper">
+        <div class="nav__logo">
+          <a href="#home"
+            class="nav__link">
+            <img class="logo--img"
+                 src="./images/logo.png"
+                 alt="logo">
+          </a>
+        </div>
```

Comment:
No need in those extra wrappers, the `nav__link` and `nav` can be direct children of `navigation`. The same for `nav__logo`.